client/src/main/HomePage/Home.jsx

```diff
@@ -1,19 +1,34 @@
+import { useState, useEffect } from 'react'
+import { useNavigate } from 'react-router'
 
 import Header from '/src/main/Header'
 import Home_News from './Home_News'
 import Home_Clips from './Home_Clips'
 import Home_Blogs from './Home_Blogs'
 import Footer from '/src/main/Footer'
 
-import { WEEKDAYS } from '/src//utils/weekdays'
+import { WEEKDAYS } from '/src/utils/constants'
+import * as user from '/src/utils/userUtils'
```

Comment:
We should rename the file to `UserUtils` (starting with upper case) and also import it like that (with same naming) when using it in files. Otherwise `user` can be confused as a variable instead of a functions library that we are using. 

Also, unless you foresee that you'll be using a lot of functions from this file, I'd suggest only importing the function that you are using, in this case seems like you are only using 1 so it makes sense to just import that function.